azext_edge/edge/providers/orchestration/resources/connector/opcua/certs.py

```diff
@@ -5,9 +5,9 @@
 # ----------------------------------------------------------------------------------------------
 
 from datetime import datetime, timezone
-import os
 import re
 from cryptography import x509
+import pathlib
```

Comment:
nit: do we need all of `pathlib` here? looks like you're only using the `Path` module
```suggestion
from pathlib import Path
```